loopchain/launcher.py

```diff
@@ -198,11 +198,11 @@ def start_as_tool(args, quick_command):
     print_prologue()
 
     try:
-        from _tools.loopchain_private_tools.demotool import DemoTool
+        from loopchain_tools.tools.basictool import BasicTool
```

Comment:
Did the new path to be applied to the tools project before here?

Reply:
Yes,  There is loopchain_tools public(github) version. see. https://github.com/icon-project/loopchain_tools